chapters/operatorsandexpressions.tex

```diff
@@ -1479,6 +1469,20 @@ \subsection{Discrete-Time Expressions}\label{discrete-time-expressions}
 functions do not become active between events.
 \end{nonnormative}
 
+For a scalar equation \lstinline!expr1 = expr2! where neither expression is of base type \lstinline!Real!, both expressions must be discrete-time expressions.
+
```

Comment:
Should restore:
```suggestion
For record
equations the equation is split into basic types before applying this
test.
```

Reply:
I tried something similar, but with a formulation that is more in the style of the first sentence, and referring to this as a _rule_ rather than a _test_ for consistency with the non-normative text below:
> For a record equation, the same rule applies to the scalar elements of the equation.